Aptoide/aptoide-client-v8
# Feature/asv 449 promote aptoide products

**What does this PR do?**

   Ads aptoide product cards in my profile. Each card, when clicked, will lead the user to the appropriate screen (app view in case of Uploader or Backup, or a website in case of TV)

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] NewAccountFragment.java
- [ ] NewAccountPresenter.java
- [ ] NewAccountNavigator.java
- [ ] NewAccountView.java
- [ ] fragment_my_account.xml
- [ ] my_account_product_cardview.xml
- [ ] AndroidManifest.xml

**How should this be manually tested?**

  Follow the use cases in Jira. Also, test if there is no problem with the deep links:
 - Clicking in AptoideTv product should direct the user to a Custom Chrome Tab. It should NOT prompt the user to open the link using Aptoide app.

Test if the following deep links are working:
- [](http://en.aptoide.com/)
- [](https://acespeeder3.en.aptoide.com/)
- [](https://en.aptoide.com/thank-you?app_id=37109754&store_name=dwiko-lukito)
- [](https://en.aptoide.com/store/rmota)
- [](https://en.aptoide.com/store/apps/group/group-10649)

 See: [](https://aptoide.atlassian.net/wiki/spaces/AN/pages/1749903/Deeplinks )

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-414](https://aptoide.atlassian.net/browse/ASV-414)

**Code Review Checklist**
